plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -96,21 +96,6 @@ public static function get_all() {
 			),
 		);
 
-		if ( Features::is_enabled( 'disable-core-profiler-fallback' ) ) {
```

Comment:
Should also remove this `Features` import: `use Automattic\WooCommerce\Admin\Features\Features;` as this was added as part of the original PR.